Source/WebCore/Modules/mediasession/MediaSession.cpp

```diff
@@ -268,7 +270,12 @@ void MediaSession::callActionHandler(const MediaSessionActionDetails& actionDeta
 
 bool MediaSession::callActionHandler(const MediaSessionActionDetails& actionDetails, TriggerGestureIndicator triggerGestureIndicator)
 {
-    if (auto handler = m_actionHandlers.get(actionDetails.action)) {
+    RefPtr<MediaSessionActionHandler> handler;
```

Comment:
Presumably, MediaSessionActionHandler needs to be ThreadSafeRefCounted now.

Reply:
You're right. No need for ThreadSafeRefCounted. The visit function is the one that gets called on an another thread and that one doesn't increment/decrement the ref count. My bad.